### PR TITLE
add tests to actions

### PR DESCRIPTION
**Pull request by glowkeeper**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -15,9 +15,9 @@ jobs:
         with:
           node-version: '16'
       - run: npm ci
-      - run: ./node_modules/.bin/eslint src
+      - run: npm run lint
```
Comment (glowkeeper) [inline .github/workflows/main.yml:18]: Better to call the script than the linter directly - what if we changed the linter?

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -15,9 +15,9 @@ jobs:
...
 
-  notify:
+  deploy:
```
Comment (glowkeeper) [inline .github/workflows/main.yml:20]: Better name - it's deploying!

File: .github/workflows/pull_request.yml [hunk 3]
```diff
@@ -12,4 +12,15 @@ jobs:
         with:
           node-version: '16'
       - run: npm ci
-      - run: ./node_modules/.bin/eslint src
+      - run: npm run lint
+
+  test:
```
Comment (glowkeeper) [inline .github/workflows/pull_request.yml:17]: added a test that's dependent on the linter...

**Review by glowkeeper — COMMENTED** [review]
Approve

**Review by vherus — APPROVED** [review]

**Pull request merged**